README.md

```diff
@@ -8,18 +8,18 @@ Please note that `teleporter-token-bridge` is still under active development and
 
 Teleporter token bridge is an application that allows users to transfer tokens between Subnets. The bridge is a set of smart contracts that are deployed across multiple Subnets, and leverages [Teleporter](https://github.com/ava-labs/teleporter) for cross-chain communication.
 
-Each bridge instance consists of one "hub" contract and at least one but possibly many "spoke" contracts. Each hub contract instance manages one asset to be bridged out to spoke instances. The hub contract lives on the Subnet where the asset to be bridged exists and locks that asset as collateral to be bridged to other Subnets. The spoke contracts, each of which has a single specified hub contract, live on other Subnets that want to import the asset bridged by their specified hub. The token bridges are designed to be permissionless: anyone can register compatible spoke instances to allow for bridging tokens from the hub instance to that new spoke. The hub contract keeps track of token balances bridged to each spoke instance, and it handles returning the original tokens back to the user when assets are bridged back to the hub instance. Spoke instances are registered with their hub contract via a Teleporter message upon creation.
+Each bridge instance consists of one "home" contract and at least one but possibly many "remote" contracts. Each home contract instance manages one asset to be bridged out to remote instances. The home contract lives on the Subnet where the asset to be bridged exists and locks that asset as collateral to be bridged to other Subnets. The remote contracts, each of which has a single specified home contract, live on other Subnets that want to import the asset bridged by their specified home. The token bridges are designed to be permissionless: anyone can register compatible remote instances to allow for bridging tokens from the home instance to that new remote instance. The home contract keeps track of token balances bridged to each remote instance, and handles returning the original tokens back to the user when assets are bridged back to the home instance. Remote instances are registered with their home contract via a Teleporter message upon creation.
```

Comment:
Piggybacking on Bernard's suggestion to clarify "remote/home _instance_", I'd take that a step further and refer to the contracts directly as `TokenHome` and `TokenRemote`. There are a few places where we use the same terminology to refer to the blockchain that the respective contract is deployed on. Because of that, in combination with other overloaded uses of the terms "home" and "remote", I think we should be specify the contracts themselves wherever possible.